### PR TITLE
new icon: processing (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by TheFoxarmy**
Addresses issue #716
I used this image to create an SVG from scratch:
image
I'm not sure how to optimize the SVG too well since it has a lot of objects.
If the original is too busy, we can just use the plain version, though.
image
For the text versions, I've used black text, but I can make it white if preferred.

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
Hi @TheFoxarmy,

Just a few things that I notice:
- The `wordmark` SVGs are named `text`. Please rename them to `wordmark` instead.
- Please ensure that you open a PR into `develop` and not `master` for future PR.
- Make sure that the PR name matches the standard laid out [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#overview-on-submitting-icons)

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
As @Thomas-Boi said, please rename the `processing-plain-text.svg` file to `processing-plain-wordmark.svg`. Also rename the `processing-original-text.svg` file to `processing-original-wordmark.svg`.

File: icons/processing/processing-plain-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+         id="path723220" />
+      <path
+         fill="#000"
+         d="m 165.624,34.88 c -4.751,0 -7.667,-1.8 -7.667,-5.292 0,-1.872 1.26,-3.6 2.736,-4.318 -1.08,-0.541 -1.764,-1.477 -1.764,-2.736 0,-1.368 0.863,-2.627 2.086,-3.167 -1.619,-0.9 -2.627,-2.664 -2.627,-5.003 0,-3.744 2.7,-6.875 8.1,-6.875 1.15,0 2.16,0.144 3.131,0.396 h 6.047 v 3.6 h -2.412 c 0.54,0.54 0.9,1.368 0.9,2.375 0,3.888 -2.7,6.551 -7.883,6.551 -0.541,0 -1.152,-0.036 -1.656,-0.145 -0.576,0.252 -0.828,0.721 -0.828,1.152 0,0.828 0.684,1.116 2.771,1.116 h 2.771 c 3.888,0 6.155,1.8 6.155,5.184 0.002,3.886 -3.884,7.162 -9.86,7.162 z m 2.448,-8.711 h -3.131 c -0.252,0 -0.648,0 -0.864,0 -1.009,0.576 -1.583,1.548 -1.583,2.556 0,1.656 1.512,2.627 3.814,2.627 2.988,0 4.607,-1.26 4.607,-3.022 0.001,-1.225 -0.863,-2.161 -2.843,-2.161 z m -1.763,-15.334 c -2.197,0 -3.529,1.332 -3.529,3.239 0,2.016 1.261,3.204 3.529,3.204 2.088,0 3.455,-1.152 3.455,-3.167 0,-2.015 -1.224,-3.276 -3.455,-3.276 z"
+         id="path723222" />
+    </g>
+  </g>
+</svg>
```
Comment (Panquesito7) [inline icons/processing/processing-plain-wordmark.svg:91]:
```suggestion
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="M64 0a50.912 50.912 0 0 0-50.912 50.912A50.912 50.912 0 0 0 64 101.824a50.912 50.912 0 0 0 50.912-50.912A50.912 50.912 0 0 0 64 0zm7.881 17.676c5.624-.011 11.595 1.931 12.595 8.304.568 3.616.954 9.624-7.923 14.186 6.873 1.857 9.17 6.28 9.25 10.631.188 10.217-9.73 16.465-18.658 15.468l.066-5.156c7.905.907 11.744-4.246 11.915-9.086.175-4.96-4.17-8.382-11.915-8.23l-.12-4.85c9.697-.718 12.428-7.432 11.221-11.161-1.653-5.106-6.929-6.049-11.593-4.253l-.797 5.687-3.71-.12.318-9.447c1.818-1.043 5.503-1.965 9.351-1.973zM61.215 39.151v38.244c2.036-.147 4.784.56 7.273.958v4.19l-22.072-.013v-4.252c2.378-.327 4.575-.87 7.175-.93V44.526c-2.928-.05-5.683-.692-7.07-1.169v-4.08l14.695-.125z" style="stroke-width:.210478"/><path d="M12.038 116.27c-.287 0-.647 0-1.031-.048v3.478c.72.048 1.463.168 2.087.289v1.943H5.802v-1.943a17.75 17.75 0 0 1 2.087-.289v-11.729a17.675 17.675 0 0 1-2.087-.288v-1.943h1.872c.767 0 2.878-.096 4.557-.096 3.958 0 6.14 1.847 6.14 5.013 0 3.142-2.398 5.613-6.333 5.613zm.05-8.084c-.385 0-.72 0-1.08.024v5.397c.311.048.623.072.911.072 1.895-.024 3.214-1.08 3.214-2.83 0-1.631-.911-2.663-3.046-2.663zm15.976 4.582c-2.447-.456-3.55.983-3.55 4.293v2.638c.719.048 1.463.168 2.086.288v1.944h-7.22v-1.944c.624-.12 1.367-.239 2.086-.288v-7.338a23.93 23.93 0 0 0-2.086-.24v-1.92c1.511-.191 3.358-.263 4.965-.215 0 .623-.072 1.582-.216 2.423h.048c.576-1.512 1.728-2.831 4.03-2.663zm6.885 9.403c-3.382 0-5.565-1.75-5.565-5.997 0-3.717 2.47-6.404 6.093-6.404 3.382 0 5.565 1.872 5.565 5.973 0 3.742-2.519 6.428-6.093 6.428zm.312-9.81c-1.368 0-2.71 1.055-2.71 3.501 0 2.447.887 3.694 2.662 3.694 1.511 0 2.663-1.223 2.663-3.646-.001-2.302-.816-3.55-2.615-3.55zm13.122 9.787c-3.478 0-5.71-1.727-5.71-5.782 0-3.718 1.92-6.572 6.285-6.572.912 0 1.919.144 2.782.407v3.838h-1.967c-.096-.576-.192-1.103-.24-1.583-.287-.096-.647-.12-.983-.12-1.655 0-2.687 1.367-2.687 3.622 0 2.182.816 3.526 3.214 3.526.84 0 1.8-.192 2.519-.528l.216 2.638c-.982.313-2.062.554-3.43.554zm15.52-5.686h-7.387c-.024 2.231 1.08 3.31 3.286 3.31 1.175 0 2.447-.264 3.478-.744l.288 2.375c-1.271.504-2.782.767-4.27.767-3.79 0-5.9-1.895-5.9-6.093 0-3.646 2.015-6.333 5.589-6.333 3.478 0 5.013 2.375 5.013 5.325 0 .41-.025.89-.097 1.393zm-5.012-4.461c-1.272 0-2.16.935-2.327 2.375h4.39C61 112.888 60.185 112 58.89 112zm10.291 10.17c-.911 0-2.135-.12-3.118-.336v-3.574h1.944c.12.48.192 1.033.24 1.56.335.072.815.12 1.2.12 1.223 0 2.062-.527 2.062-1.415 0-2.087-5.757-.984-5.757-4.941 0-2.16 1.752-3.814 5.133-3.814.888 0 2.015.12 3.095.36v3.43h-1.943c-.12-.48-.192-1.032-.24-1.536a5.72 5.72 0 0 0-1.08-.12c-1.295 0-1.895.528-1.895 1.32 0 2.11 5.757 1.127 5.757 4.87 0 2.325-2.088 4.076-5.398 4.076zm10.532 0c-.912 0-2.135-.12-3.119-.336v-3.574h1.944c.12.48.192 1.033.24 1.56.336.072.815.12 1.199.12 1.223 0 2.064-.527 2.064-1.415 0-2.087-5.758-.984-5.758-4.941 0-2.16 1.752-3.814 5.133-3.814.888 0 2.015.12 3.096.36v3.43H82.57c-.12-.48-.192-1.032-.24-1.536a5.714 5.714 0 0 0-1.08-.12c-1.295 0-1.895.528-1.895 1.32 0 2.11 5.757 1.127 5.757 4.87 0 2.325-2.088 4.076-5.398 4.076zm6.886-.24v-1.943c.623-.12 1.367-.24 2.086-.289v-7.338a23.9 23.9 0 0 0-2.087-.24v-1.92c1.512-.191 3.358-.263 5.133-.215v9.714c.672.048 1.367.168 1.944.289v1.943h-7.077zm3.598-13.456a1.88 1.88 0 0 1-1.896-1.871c0-1.008.84-1.848 1.896-1.848 1.054 0 1.918.816 1.918 1.848 0 1.031-.863 1.87-1.918 1.87zm13.937 5.828c0-1.247-.336-1.99-1.464-1.99-1.56 0-2.783 1.775-2.783 3.861v3.527c.576.048 1.224.12 1.752.264v1.966h-6.885v-1.943c.625-.12 1.368-.24 2.087-.288v-7.34a23.917 23.917 0 0 0-2.087-.24v-1.919c1.512-.191 3.358-.263 4.965-.215 0 .551-.047 1.51-.168 2.135l.025.024c.72-1.416 2.11-2.4 4.053-2.4 2.686 0 3.55 1.728 3.55 3.814v6.141c.696.048 1.391.168 1.967.289v1.943h-5.013v-7.629zM115.506 128c-3.166 0-5.11-1.2-5.11-3.527 0-1.247.84-2.399 1.824-2.877-.72-.36-1.176-.984-1.176-1.823 0-.912.575-1.75 1.39-2.11-1.079-.6-1.75-1.776-1.75-3.335 0-2.495 1.799-4.581 5.398-4.581.766 0 1.439.096 2.086.264h4.03v2.399h-1.608c.36.36.6.911.6 1.583 0 2.59-1.8 4.365-5.253 4.365-.36 0-.768-.024-1.104-.097-.383.168-.551.48-.551.768 0 .552.455.744 1.846.744h1.847c2.59 0 4.101 1.2 4.101 3.454.002 2.59-2.588 4.773-6.57 4.773zm1.631-5.805h-2.662c-.673.384-1.055 1.032-1.055 1.703 0 1.104 1.008 1.751 2.542 1.751 1.99 0 3.07-.84 3.07-2.014 0-.816-.575-1.44-1.895-1.44zm-1.175-10.218c-1.464 0-2.351.887-2.351 2.158 0 1.344.84 2.135 2.351 2.135 1.392 0 2.303-.767 2.303-2.11s-.816-2.183-2.303-2.183z"/></svg>

```

File: icons/processing/processing-original-wordmark.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....197-2.987-2.34 0-4.176 2.664-4.176 5.795v5.292c0.864 0.072 1.836 0.18 2.629 0.396v2.951h-10.332v-2.916c0.937-0.18 2.053-0.359 3.132-0.432v-11.014c-0.864-0.144-2.087-0.288-3.132-0.36v-2.88c2.269-0.288 5.039-0.396 7.451-0.324 0 0.828-0.071 2.268-0.252 3.204l0.037 0.036c1.079-2.124 3.166-3.6 6.082-3.6 4.031 0 5.327 2.592 5.327 5.723v9.215c1.044 0.072 2.088 0.252 2.952 0.433v2.916h-7.523v-11.448z"/><path d="m165.62 34.88c-4.751 0-7.667-1.8-7.667-5.292 0-1.872 1.26-3.6 2.736-4.318-1.08-0.541-1.764-1.477-1.764-2.736 0-1.368 0.863-2.627 2.086-3.167-1.619-0.9-2.627-2.664-2.627-5.003 0-3.744 2.7-6.875 8.1-6.875 1.15 0 2.16 0.144 3.131 0.396h6.047v3.6h-2.412c0.54 0.54 0.9 1.368 0.9 2.375 0 3.888-2.7 6.551-7.883 6.551-0.541 0-1.152-0.036-1.656-0.145-0.576 0.252-0.828 0.721-0.828 1.152 0 0.828 0.684 1.116 2.771 1.116h2.771c3.888 0 6.155 1.8 6.155 5.184 2e-3 3.886-3.884 7.162-9.86 7.162zm2.448-8.711h-3.131-0.864c-1.009 0.576-1.583 1.548-1.583 2.556 0 1.656 1.512 2.627 3.814 2.627 2.988 0 4.607-1.26 4.607-3.022 1e-3 -1.225-0.863-2.161-2.843-2.161zm-1.763-15.334c-2.197 0-3.529 1.332-3.529 3.239 0 2.016 1.261 3.204 3.529 3.204 2.088 0 3.455-1.152 3.455-3.167s-1.224-3.276-3.455-3.276z"/></g></svg>
```
Comment (Panquesito7) [inline icons/processing/processing-original-wordmark.svg:1]:
```suggestion
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="a" x1="64" x2="64" y1="169" y2="297" gradientTransform="translate(-.555 -2.02) scale(1.0087)" gradientUnits="userSpaceOnUse"><stop stop-color="#fff" stop-opacity=".918" offset="0"/><stop stop-color="#fff" stop-opacity=".559" offset="1"/></linearGradient></defs><g transform="translate(13.105 -134.42) scale(.79538)"><circle cx="64" cy="233" r="64"/><path d="m55.017 180.15 23.042 11.422" fill="#6e8375" stroke="#123859"/><path d="m67.823 186.63 9.13 4.585" fill="#6e8375" stroke="#fff" stroke-width=".5"/><path d="m7.21 220.26 31.684 20.902" fill="none" stroke="#152e44" stroke-width=".65"/><path d="M4.1 211.87c2.005 2.786 8.338 10.643 10.64 12.985" fill="none" stroke="#e5b828" stroke-width=".5"/><path d="m126.79 222.99-19.683 5.138" fill="#0f212e" stroke="#1b344c" stroke-width=".3"/><path d="m124.68 212.54-20.638 28.575-12.435-29.633" fill="none" stroke="#326a9c" style="font-variation-settings:normal"/><path d="m89.88 208.57 12.521 30.631" fill="none" stroke="#e5b828" stroke-width=".514"/><path d="m73.966 274.73 29.334-33.297" fill="#1b344b" stroke="#326a9c" stroke-width=".6"/><path d="m83.873 249.96 15.214 25.532" fill="#326a9c" stroke="#326a9c" stroke-width=".8" style="font-variation-settings:normal"/><path d="m85.53 293.66-11.383-18.655" fill="none" stroke="#19344e" stroke-width=".6"/><g fill="#0a2035"><g stroke="#0a2035"><path d="m99.086 275.49-24.672 10.385" stroke-width=".8" style="font-variation-settings:normal"/><path d="m62.045 291.24 12.369-5.358" stroke-width=".4" style="font-variation-settings:normal"/><path d="m106.89 280.52-7.938-4.366" stroke-width=".8" style="-inkscape-stroke:none;font-variation-settings:normal"/></g><path d="m99.219 275.63 22.357-51.197" stroke="#10273c" stroke-width=".8" style="-inkscape-stroke:none;font-variation-settings:normal"/></g><g fill="none"><path d="m38.633 242.29-19.785 27.853" stroke="#152e44" stroke-width=".64"/><path d="m18.785 236.27 19.645 4.763" stroke="#152e44" stroke-width=".8"/><g stroke-width=".5"><path d="m21.799 235.57 13.135 37.394" stroke="#6e8475"/><path d="m11.462 269.19 10.355-34.267" stroke="#d0d0d0"/><path d="m35.438 273.13-3.745-36.947" stroke="#33526d"/></g><path d="m18.19 214.77 20.968 26.26" stroke="#152e44" stroke-width=".65"/><path d="m6.548 224.36 12.568 11.245" stroke="#0c2135" stroke-width=".61"/><path d="m.356 230.58 25.335-8.695" stroke="#e5b828" stroke-width=".5"/><path d="m.277 230.18 29.065-10.038" stroke="#d0d0d0" stroke-width=".5"/><path d="m22.414 234.23 6.608-13.51-6.608 13.51" stroke="#e5b828" stroke-width=".6"/><path d="M3.857 212.07 21.8 234.45" stroke="#d0d0d0" stroke-width=".5"/><path d="m29.292 220.23 6.034 50.333" stroke="#d9d9d9" stroke-width=".5"/></g><path d="M35.885 273.97v.447L22.973 237.47" fill="#6e8375" stroke="#3c5b75" stroke-width=".5"/><g fill="none"><g stroke="#152e44"><path d="m38.894 241.5 3.44 15.081" stroke-width=".65"/><path d="m42.598 256.84 2.249 9.459 5.564 29.052m-11.12-53.461 32.676-8.07" stroke-width=".55"/></g><path d="m43.903 236.08 29.998 38.439" stroke="#19344e" stroke-width=".748"/></g><path d="M41.894 272.75v-5.345c2.99-.41 5.751-1.094 9.02-1.17v-41.26c-3.68-.062-7.144-.87-8.886-1.47v-5.13l18.472-.155v48.074c2.559-.185 6.013.705 9.142 1.205v5.268z" fill="#fff"/><path d="m61.78 183.95 7.938 33.734" fill="none" stroke="#081a26" stroke-width=".714"/><path d="m114.95 204.66-28.507-9.307" fill="#6e8375" stroke="#0f2c48" stroke-width=".5"/><path d="m61.752 205.58.4-11.877c5.627-3.227 25.472-5.534 27.589 7.959.713 4.546 1.198 12.097-9.96 17.833 8.64 2.334 11.528 7.893 11.628 13.363.236 12.844-12.233 20.698-23.456 19.444l.084-6.482c9.937 1.14 14.763-5.336 14.978-11.42.22-6.236-5.241-10.537-14.978-10.346l-.15-6.096c12.189-.904 15.623-9.344 14.105-14.031-2.078-6.419-8.71-7.604-14.573-5.346l-1.003 7.15z" fill="#fff"/><g fill="none"><path d="m93.511 289.92-17.575-13.9" stroke="#19344e" stroke-width=".95"/><g stroke="#0b1620"><path d="m61.453 233.83 8.824-.189" stroke-width=".5"/><path d="m61.946 229.52 8.235 2.712" stroke-width=".748"/><path d="m71.206 224.74 1.178 6.098" stroke-width=".713"/><path d="m77.314 227.18-2.9 3.835" stroke-width=".348"/><path d="m74.04 232.84 4.022-4.911" stroke-width=".248"/><path d="m75.911 232.42 5.66-1.216" stroke-width=".348"/></g><path d="m9.027 252.87 9.76-16.54" stroke="#0c2135" stroke-width=".825"/><path d="m18.661 237.78-2.926 16.75" stroke="#0c2135" stroke-width=".41"/><path d="m-.01 236.01 21.698-1.44" stroke="#e5b828" stroke-width=".4"/><path d="m.299 236.4 19.18-1.231" stroke="#78a657" stroke-width=".5"/><path d="m4.53 210.06 10.42 3.121" stroke="#0c2135" stroke-width=".31"/><path d="m17.132 215.1 1.323 11.642" stroke="#081725" stroke-width=".31"/><path d="m84.961 218.08 19.899-26.271" stroke="#0f2c48" stroke-width=".714"/><g stroke="#123859"><path d="m84.205 193.41 7.604-11.64" stroke-width=".814"/><path d="m65.484 181.96 29.567 1.852" stroke-dasharray="6.51201, 0.814001" stroke-width=".814"/><path d="m62.437 182.2 4.716.268" stroke-dasharray="2.08783, 2.08783" stroke-width=".348"/><path d="m61.383 181.7-33.999 23.945" stroke-width=".714"/></g><path d="m53.715 179.4 14.086 7.21" stroke="#e5b828" stroke-width=".714"/><path d="m53.38 179.17-22.246 37.729" stroke="#e5b828" stroke-width=".314"/><path d="m29.289 220.57 19.03-31.926" stroke="#d0d0d0" stroke-width=".514"/><path d="m55.959 175.48 6.218 6.085" stroke="#081a26" stroke-width=".714"/><path d="m68.6 174.94-6.692 7.201" stroke="#173149" stroke-width=".714"/></g><path d="m115.06 204.05-28.479-9.139" fill="#6e8375" stroke="#fff" stroke-width=".7"/><path d="m27.153 278.09 9.21-3.439" fill="#6e8375" stroke="#11385e" stroke-width=".6"/><g fill="none"><g stroke-width=".714"><path d="m36.332 274.98 7.56-38.65m-7.779 38.86 36.551-.329" stroke="#d9d9d9"/><path d="m46.754 171.64 7.087 7.607" stroke="#e5b828"/></g><path d="m52.455 181.45-4.248 7.378" stroke="#123859" stroke-width=".514"/></g><path d="m54.386 179.56 29.857-6.666" fill="#06111b" stroke="#e5b828" stroke-width=".514"/><g fill="none"><path d="m6.627 205 19.38 13.084" stroke="#e5b828" stroke-width=".4"/><path d="m6.65 204.61 22.598 15.217" stroke="#d0d0d0" stroke-width=".4"/><g stroke-width=".5"><path d="m11.857 269.27 8.853-29.445" stroke="#79a6cc"/><path d="m21.778 234.65 21.857 1.265" stroke="#fff"/><path d="m21.975 235.09 20.296 1.255" stroke="#32556e"/><path d="m15.162 225.42 6.581 8.589" stroke="#e5b828"/><path d="m14.7 225.62 7.21 8.943" stroke="#fff"/></g></g><g fill="#6e8375"><path d="m29.609 220.46 14.03 14.756" stroke="#405c74" stroke-width=".5"/><path d="m43.518 235.97-14.174-15.013" stroke="#fff" stroke-width=".6"/><path d="m21.274 280.9 13.309-4.953" stroke="#c4c32d" stroke-width=".6"/></g><path d="m78.811 241.49-4.11-6.561" fill="#0f212e" stroke="#11385e" stroke-width=".3"/><circle cx="64" cy="233" r="61.336" fill="none" stroke="url(#a)" stroke-width="5.328"/></g><path d="M12.038 116.274c-.287 0-.647 0-1.031-.048v3.478c.72.048 1.463.168 2.087.288v1.944H5.802v-1.944c.624-.12 1.368-.24 2.087-.288v-11.73a17.675 17.675 0 0 1-2.087-.287v-1.943h1.872c.767 0 2.878-.096 4.557-.096 3.958 0 6.14 1.846 6.14 5.013 0 3.142-2.398 5.613-6.333 5.613zm.05-8.084c-.385 0-.72 0-1.08.024v5.397c.311.048.623.072.911.072 1.895-.024 3.214-1.08 3.214-2.83 0-1.632-.911-2.663-3.046-2.663zm15.976 4.581c-2.447-.455-3.55.984-3.55 4.294v2.638c.719.048 1.463.168 2.086.288v1.943h-7.22v-1.943c.624-.12 1.368-.24 2.087-.288v-7.339a23.931 23.931 0 0 0-2.087-.24v-1.919c1.511-.192 3.358-.264 4.965-.216 0 .624-.072 1.583-.216 2.423h.048c.576-1.511 1.728-2.83 4.03-2.663zm6.885 9.404c-3.382 0-5.565-1.75-5.565-5.997 0-3.718 2.47-6.404 6.093-6.404 3.382 0 5.565 1.871 5.565 5.973 0 3.742-2.519 6.428-6.093 6.428zm.312-9.81c-1.368 0-2.71 1.055-2.71 3.501s.887 3.694 2.662 3.694c1.511 0 2.663-1.224 2.663-3.646 0-2.302-.816-3.55-2.615-3.55zm13.122 9.786c-3.478 0-5.71-1.727-5.71-5.781 0-3.718 1.92-6.573 6.285-6.573.912 0 1.919.144 2.783.408v3.838h-1.968c-.095-.576-.191-1.104-.24-1.584-.287-.095-.647-.12-.983-.12-1.655 0-2.687 1.367-2.687 3.622 0 2.183.816 3.527 3.214 3.527.84 0 1.8-.193 2.519-.529l.216 2.639c-.982.312-2.062.553-3.43.553zm15.521-5.685h-7.388c-.024 2.23 1.08 3.31 3.286 3.31 1.175 0 2.447-.264 3.479-.744l.287 2.374c-1.271.504-2.782.767-4.27.767-3.789 0-5.9-1.895-5.9-6.092 0-3.647 2.015-6.333 5.589-6.333 3.478 0 5.013 2.374 5.013 5.325 0 .41-.024.889-.096 1.393zm-5.013-4.462c-1.271 0-2.16.936-2.327 2.375h4.39c.047-1.487-.768-2.375-2.063-2.375zm10.292 10.171c-.912 0-2.135-.12-3.118-.336v-3.574h1.943c.12.48.192 1.032.24 1.56.336.071.815.12 1.2.12 1.223 0 2.062-.528 2.062-1.415 0-2.088-5.757-.984-5.757-4.942 0-2.159 1.752-3.813 5.134-3.813.887 0 2.014.12 3.094.36v3.43h-1.943c-.12-.48-.192-1.032-.24-1.536a5.72 5.72 0 0 0-1.08-.12c-1.295 0-1.894.528-1.894 1.319 0 2.11 5.757 1.127 5.757 4.87-.001 2.325-2.088 4.077-5.398 4.077zm10.529 0c-.912 0-2.135-.12-3.119-.336v-3.574h1.943c.12.48.193 1.032.24 1.56.337.071.816.12 1.2.12 1.223 0 2.063-.528 2.063-1.415 0-2.088-5.757-.984-5.757-4.942 0-2.159 1.752-3.813 5.133-3.813.888 0 2.014.12 3.095.36v3.43h-1.943c-.12-.48-.192-1.032-.24-1.536a5.714 5.714 0 0 0-1.08-.12c-1.294 0-1.894.528-1.894 1.319 0 2.11 5.756 1.127 5.756 4.87 0 2.325-2.087 4.077-5.397 4.077zm6.89-.24v-1.943a17.75 17.75 0 0 1 2.087-.289v-7.339a23.9 23.9 0 0 0-2.087-.24v-1.919c1.512-.192 3.358-.264 5.133-.216v9.715a15.12 15.12 0 0 1 1.944.288v1.944h-7.077zm3.599-13.457a1.88 1.88 0 0 1-1.896-1.87c0-1.008.84-1.848 1.896-1.848 1.054 0 1.918.816 1.918 1.847s-.863 1.871-1.918 1.871zm13.934 5.829c0-1.248-.336-1.99-1.464-1.99-1.56 0-2.783 1.774-2.783 3.86v3.527c.576.048 1.223.12 1.752.264v1.967h-6.885v-1.943c.624-.12 1.368-.24 2.087-.288v-7.34a23.918 23.918 0 0 0-2.087-.24v-1.919c1.512-.192 3.358-.264 4.965-.216 0 .552-.047 1.512-.168 2.135l.025.024c.719-1.415 2.11-2.399 4.053-2.399 2.686 0 3.55 1.728 3.55 3.814v6.14c.695.049 1.391.169 1.967.29v1.943h-5.013v-7.63zm11.369 13.697c-3.167 0-5.11-1.2-5.11-3.527 0-1.247.84-2.399 1.824-2.877-.72-.36-1.176-.985-1.176-1.824 0-.911.575-1.75 1.39-2.11-1.079-.6-1.75-1.775-1.75-3.334 0-2.495 1.799-4.581 5.397-4.581.767 0 1.44.096 2.087.263h4.03v2.4h-1.608c.36.36.6.911.6 1.582 0 2.591-1.8 4.366-5.253 4.366-.36 0-.768-.024-1.104-.097-.384.168-.552.48-.552.768 0 .552.456.743 1.847.743h1.846c2.591 0 4.102 1.2 4.102 3.455.001 2.59-2.588 4.773-6.57 4.773zm1.63-5.805h-2.66c-.673.384-1.055 1.031-1.055 1.703 0 1.104 1.007 1.75 2.541 1.75 1.991 0 3.07-.839 3.07-2.013.001-.816-.575-1.44-1.894-1.44zm-1.174-10.219c-1.464 0-2.352.888-2.352 2.159 0 1.343.84 2.135 2.352 2.135 1.391 0 2.302-.768 2.302-2.11s-.815-2.184-2.302-2.184z"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
These suggestions optimize the SVGs for less size with the same quality.

File: devicon.json [hunk 3]
```diff
@@ -3046,6 +3046,31 @@
...
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "fonts": [
```
Comment (Panquesito7) [inline devicon.json:3066]:
```suggestion
            "font": [
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution, @TheFoxarmy! 👍 🎉

**Comment by Thomas-Boi** [comment]
This PR looks good. Great job @TheFoxarmy!

**Pull request merged**